.github/workflows/coverage-check.yml

```diff
@@ -0,0 +1,68 @@
+name: Check Test Coverage
+
+on:
+  pull_request:
+    branches: [development]
+
+jobs:
+  test-coverage:
+    name: Check Test Coverage
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+
+    steps:
+      - name: Checkout development branch
+        uses: actions/checkout@v3
+        with:
+          ref: development
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+
+      - name: Install dependencies
+        run: sudo apt -y update && sudo apt -y install libusb-1.0-0-dev libudev-dev
+
+      - name: Yarn Install
+        run: yarn install --mode=skip-build && yarn allow-scripts
+
+      - name: Setup Foundry
+        uses: onbjerg/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      - name: Setup Forge
+        run: yarn workspace @ubiquity/contracts forge:install
+
+      - name: Get development branch coverage
+        id: coverage-development
+        run: |
+          cd packages/contracts
+
+          COVERAGE_DEVELOPMENT_OUTPUT=$(forge coverage --no-match-path=**/utils/**=**/utils/**)
```

Comment:
pls remove the `--no-match-path=**/utils/**=**/utils/**` option

Reply:
done